Daf.Meta/Layers/StagingTable.cs

```diff
@@ -8,8 +8,10 @@ namespace Daf.Meta.Layers
 {
 	public class StagingTable
 	{
-		[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "We need at least an init setter in order to support deserialization.")]
-		public List<StagingColumn> Columns { get; init; } = new List<StagingColumn>();
+		// Deserializing doesn't actually set this, it only adds new columns.
+		// Without a setter the application crashes when loading new data due to the way metadata is refreshed.
```

Comment:
Can't recall exactly why I needed to change from init to set and I tried changing it back but I can't make it crash. Possibly since I made changes in Model.cs that made including a setter redundant. Changing it back to init for now.

I'm planning to overhaul how data is loaded/saved and serialized after we've fixed dependency injection so that we're able to have unit tests in place when I do.